controllers/registry.go

```diff
@@ -225,9 +226,11 @@ func (r *DPAReconciler) getProviderSecret(secretName string) (corev1.Secret, err
 	if err != nil {
 		return secret, err
 	}
-	r.Log.Info(fmt.Sprintf("got provider secret name: %s", secret.Name))
```

Comment:
This log does not seem to be that important and does not warrant log level verbosity changes as well, IMO lets remove this log as well as the log level changes.